### PR TITLE
Items per page

### PR DESCRIPTION
**Pull request by kotsios5**
https://github.com/open-classifieds/openclassifieds2/issues/1628

### CODE REVIEW

File: themes/default/views/pages/ad/listing.php [hunk 1]
```diff
@@ -71,6 +71,17 @@ class="btn btn-default btn-sm">
             <li><a href="?<?=http_build_query(['sort' => 'published-desc'] + Request::current()->query())?>"><?=__('Newest')?></a></li>
             <li><a href="?<?=http_build_query(['sort' => 'published-asc'] + Request::current()->query())?>"><?=__('Oldest')?></a></li>
         </ul>
+        <div class="btn-group">
+          <button class="btn btn-default btn-sm dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <?=__('Items per page')?> <span class="caret"></span>
```
Comment (neo22s) [inline themes/default/views/pages/ad/listing.php]: Would be cool if you add here what's been selected. So they know if they are seeing 5 or 100 ads ;)

File: themes/default/views/pages/ad/listing.php [hunk 2]
```diff
@@ -71,6 +71,17 @@ class="btn btn-default btn-sm">
...
+          </button>
+          <ul class="dropdown-menu">
+            <li><a href="?<?=http_build_query(['items_per_page' => '5'] + Request::current()->query())?>"> 5 </a></li>
+            <li><a href="?<?=http_build_query(['items_per_page' => '10'] + Request::current()->query())?>"> 10 </a></li>
+            <li><a href="?<?=http_build_query(['items_per_page' => '20'] + Request::current()->query())?>"> 20 </a></li>
+            <li><a href="?<?=http_build_query(['items_per_page' => '50'] + Request::current()->query())?>"> 50 </a></li>
```
Comment (neo22s) [inline themes/default/views/pages/ad/listing.php]: add 100 too ;)

**Comment by neo22s** [comment]
I am merging so I can test it, remember we need to move it to yclas too ;)

**Comment by kotsios5** [comment]
sure!

**Comment by neo22s** [comment]
Next to the Number, maybe something like 5 items per page, or check how they name it in other sites xD

but do it tomorrow! it works good ;)

**Pull request merged**